letter/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Markup Letter</title>
+</head>
+<body>
+    <p>
+        <address class="sender-column">
+            <strong>Dr. Eleanor Gaye</strong><br>
+           Awesome Science faculty <br>
+           University of Awesome <br>
+           Bobtown, CA 99999, <br>
+           USA <br>
+           <strong>Tel: </strong> 123-456-7890 <br>
+           <strong>Email: </strong> no_reply@example.com
+        </address>   
+    </p>
+
+
+    <p class="sender-column">20 January 2016</p> 
+
+    <p>
+        <address>
+            <strong>Miss Eileen Dover</strong> <br>
+            4321 Cliff Top Edge <br>
+            Dover, CT9 XXX <br>
+            UK
+        </address>
+    </p>
+
+    <h1>Re: Eileen Dover university application</h1> 
+
+    <p>Dear Eileen,</p>
+
+    <p>Thank you for your recent application to join us at the University of Awesome's science faculty to study as part of your PhD next year. I will answer your questions one by one, in the following sections.</p> 
+
+    <h3>Starting dates</h3> 
```

Comment:
Great job using a heading tag here ✨ It is considered good accessibility practice to use them in order i.e h1 should be followed by h2 then h3. You have skipped h2 in this case :) This h3 should be a h2